addons/mail/static/src/emoji_picker/emoji_picker.js

```diff
@@ -202,16 +229,13 @@ export class EmojiPicker extends Component {
             }
             case "ArrowDown": {
                 const newIndex = this.state.activeEmojiIndex + EMOJI_PER_ROW;
-                if (newIndex <= this.getEmojis().length - 1) {
+                if (newIndex < this.itemsAmount) {
                     this.state.activeEmojiIndex = newIndex;
```

Comment:
(Bug already present before your PR)

The calculation of the `newIndex` assumes that there are always `EMOJI_PER_ROW` emojis per row, but this is necessarily not the case. For example, when navigating from one category to another, the last row of the category is often not completely filled. In this situation, the resulting offset will be incorrect because it is not calculated using the actual number of items in the row.

Reply:
Good catch, will fix it in another PR